types/xast/index.d.ts

```diff
@@ -12,18 +12,36 @@ import { Parent as UnistParent, Literal as UnistLiteral, Node as UnistNode } fro
 export { UnistNode as Node };
 
 /**
- * Node in xast containing other nodes.
- * Its content is limited to only other xast content.
+ * Node in xast containing a value.
  */
-export interface Parent extends UnistParent {
```

Comment:
I think I’m going to have to turn this into a `unist.Parent` re-export to prevent breaking changes.